extendr-api/src/wrapper/nullable.rs

```diff
@@ -77,3 +131,88 @@ where
         }
     }
 }
+
+impl<T> Into<Option<T>> for Nullable<T> {
```

Comment:
Why are we using `Into` here rather than `From`?

Reply:
No particular reason other than me not knowing the convention/best practices :)

Reply:
Yep, always use `From` instead of `Into` if you can. There are some scenarios where you can't do that though.